src/tests/bookmark-service.test.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * @file This file is used to test the bookmark service
+ * /
+import * as BookmarkService from "../services/bookmark-service";
+
+/**
```

Comment:
good work. This code is perfect. Have all the testing that we need

Reply:
Merged.